### ISSUE TITLE
Put feedback and symbol requests here!

### ISSUE THREAD

**Issue by matthewscharles** [matthewscharles/instrument-maker-symbols#8]
Please leave requests for symbols or categories in the comments here.  Feedback on clarity of the symbols would be very welcome as well.

**Comment by matthewscharles** [comment]
For requests: alternatively, you could fork this file: 
https://github.com/matthewscharles/instrument-maker-symbols/blob/master/wishlist-categories.md

(if you haven't used GitHub before, this is a great tutorial: https://www.youtube.com/watch?v=oPpnCh7InLY)

**Comment by matthewscharles** [comment]
Here's an example that was sent by direct message (@waftlord ended up contributing to the list directly in the end):

_I need laptop/computer (maybe you excluded that intentionally)

Midi interface .(could be midi and deck...)

Skoog 

Soundbeam symbols_

**Comment by IamKriss42** [comment]
Upon first examination of the symbols used i,  all looks really good to me.  The only bit I found confusing was white noise.  In my opinion it is too similar to noise. I'm wondering if there is a way of making it clearer perhaps by the use of more white that it is white noise?

**Comment by matthewscharles** [comment]
Thanks @IamKriss42 ..yeah the differences between some of these symbols are less clear, and I hadn't thought of that one!! Especially since this is quite abstract to start with. 

How about this?  I made the main noise symbol multicoloured instead..

Noise 
![image](https://user-images.githubusercontent.com/1176858/54377037-44763f80-467c-11e9-9d30-d6cdade438bf.png)

White noise
![image](https://user-images.githubusercontent.com/1176858/54377066-50620180-467c-11e9-881e-84928ff0c73a.png)

(version 1380a58e4513c310beb41c412b3dd9905815213d)

**Comment by IamKriss42** [comment]
Yes!  This is much clearer.  It is logical and I think it works.  :)
